bin/upload_sarif_to_github

```diff
@@ -31,7 +31,7 @@ sarif_base64_temp_file=$(mktemp)
 
 gzip -c "$sarif_file" | base64 > "$sarif_base64_temp_file"
 
-if [[ -n $BUILDKITE_PULL_REQUEST ]]; then
+if [[ $BUILDKITE_PULL_REQUEST != "false" ]]; then
```

Comment:
and maybe I should add

```sh
else
  exit 0
```

because we don't want to upload if it's not main branch or PR 🤔

Reply:
Just for being extra paranoid, you could cover both cases (`false` and empty/not defined) by doing something like this:

```suggestion
if [[ "${BUILDKITE_PULL_REQUEST:-false}" != "false" ]]; then
```

Not that I'd expect Buildkite to suddenly change the way they hand the `BUILDKITE_PULL_REQUEST` env var in the future, but, you know, at least we'd cover our bases.